### DISCUSSION TITLE
Seed Saving

### DISCUSSION THREAD

**Discussion by Jennacd** [agilegardenclub/public-garden#73] [General]
I thought I'd start a discussion about seed saving.

## Why support seed saving?

Seed saving is an important practice to support because:

- Seed saving supports our mission to increase community resiliency.
- We want to be attractive to seed savers and seed saving organizations.
- This will increase the accuracy of our timeline data.  We will be able to collect data on how long plants need to be in the garden in order to take advantage of this final part of the life cycle.  Seed packets only share days to maturity leaving gardeners to guess what additional time is needed for the plant to live through it's harvest stage and develop mature seeds.

## Design considerations

Different plants generate seeds along different timelines.  Some seeds mature during a completely different phase than harvesting (lettuce and other leafy greens, root crops) while others might overlap (beans, tomatoes, squash).  To represent this we have decided to add an "end harvest" date.  If the end harvest is before the pull date the timeline color will go back to green to represent that the plant is in the garden but not providing food.

The gardener will want to know about the quality of the plant that produced the seed.  By connecting the saved seeds with the planting card of the parent plant the gardener will have access to outcome data **and public observations.**  The gardener who saved the seeds can choose to add more information or not, the essential data is there.  **The gardeners can message back and forth through the app as needed to determine sharing logisitics.**

Seeds availablity will be included in Chapter and Garden Overviews.  The Chapter Overview will include a list of crops available to the chapter.  Click the crop and it will show varieties.  Click the variety and it will take you to the planting card with timeline and outcome data and the ability to connect with the gardener.  The Garden Overview will have the list of crops that the gardener has available for sharing.  Similarly by clicking the crop, then variety, you'll get to the planting card.

When the gardener has saved seeds from a plant they can check a box in the planting card to note that seeds were saved and an additional box to represent that seeds are available to the chapter.

A later feature could be a seed finder tool with search fields.  Helpful filters would be: year, family, crop, variety, and vendor/gardener.  We may find this is unnecessary if searching through the above process is easy.

Thoughts?

**Comment by philipmjohnson** [comment]
I think you pretty much nailed it. For the MVP, I propose the following subset of functionality:

* Support an optional "end-harvest" field for timeline data, provide a few real examples from the two gardens.
* Show the "second green bar" in the timeline visualization when the end harvest field is provided.
* Support an optional "seeds-available" field for a planting; if supplied and has the value true, then this indicates that seeds are available for sharing to the community.  Provide a few real examples.
* Provide realistic seed availability based on the seeds-availability field in Chapter and Garden overviews.

OK?
